rotkehlchen/chain/ethereum/modules/aave/graph.py

```diff
@@ -565,6 +639,44 @@ def _process_events(
             total_earned_liquidations=total_earned_liquidations,
         )
 
+    def _process_graph_query_result(
+        self,
+        query: Dict[str, Any],
+        deposits: List[AaveDepositWithdrawalEvent],
+        withdrawals: List[AaveDepositWithdrawalEvent],
+        borrows: List[AaveBorrowEvent],
+        repays: List[AaveRepayEvent],
+        liquidation_calls: List[AaveLiquidationEvent],
+        user_merged_data: Dict[str, Any],
+        from_ts: Timestamp,
+        to_ts: Timestamp,
+    ) -> None:
+        """
+        Given a query result from the graph (data) this function extracts
+        """
+        if 'users' in query and len(query['users']) != 0:
```

Comment:
Hmm so if this is `False` nothing happens?

Then please change this function and make it so that if this is `False` the function should exit.

This way the rest of the function will be more readable and less indented.